test_game.py

```diff
@@ -0,0 +1,150 @@
+"""The test game."""
+
+
+PAYOFF_TABLE = [[2, 3],  # payoff_table[my_move][their_move]
+                [-3, 1]
+               ]
```

Comment:
This payoff table doesn't seem to satisfy A[i][i] = 0 (but maybe relaxing this constraint could result in more interesting games)

Reply:
That was a constraint? Oops. In any case, it's just a test table.